src/sentry/static/sentry/app/stores/configStore.tsx

```diff
@@ -33,33 +36,39 @@ const configStoreConfig: Reflux.StoreDefinition & ConfigStoreInterface = {
       ...this.config,
       [key]: value,
     };
-    this.trigger({[key]: value});
```

Comment:
Why remove the arguments here? Are there store listeners that rely on this behavior?

Reply:
i removed it because it was confusing to calculate the right arguments in case the theme changes. i did check and didn't find any but i think i checked wrong. I have now checked `rg --pcre2 '(?<!import )ConfigStore(?!\.[gs]et)' -C3` and found that app/index.tsx uses it, but i think it wasn't working properly anyways (receiving config both directly and through withConfig HOC), so i've refactored it to be less roundabout. i also noticed https://github.com/getsentry/sentry/blob/75ddf03d38cd83a1845687bf3d313ad978ce0e3f/src/sentry/static/sentry/app/views/settings/account/accountDetails.tsx#L25 which i think is not correct but is now retroactively correct with this change.